salt/modules/win_pkg.py

```diff
@@ -120,8 +120,7 @@ def latest_version(*names, **kwargs):
         if name in installed_pkgs:
             log.trace('Determining latest installed version of %s', name)
             try:
-                latest_installed = sorted(
-                    installed_pkgs[name], cmp=_reverse_cmp_pkg_versions).pop()
+                latest_installed = sorted(installed_pkgs[name]).pop()
```

Comment:
Looks like it can still be done, its just implemented differently now.
https://docs.python.org/3/library/functools.html#functools.cmp_to_key
https://docs.python.org/3/library/functions.html#sorted
https://docs.python.org/3/howto/sorting.html